dom-querying/main.js

```diff
@@ -0,0 +1,13 @@
+console.log('Hello, world');
+
+console.log(document.querySelector('h1'));
+console.dir(document.getElementsByTagName('h1'));
+
+console.log(document.querySelector('#explanation'));
+console.dir(document.getElementById('explanation'));
+
+console.log(document.querySelector('.hint'));
+console.dir(document.getElementsByClassName('hint'));
+
+console.log(document.querySelectorAll('p'));
+console.log(document.querySelectorAll('.example_link'));
```

Comment:
there is a typo here, which is why the node list was empty. you want a - not a _